api/app/src/routes/fhir-proxy.ts

```diff
@@ -0,0 +1,16 @@
+import proxy from "express-http-proxy";
+
+// TODO make this dynamic/config/secret
+const router = proxy("https://fhir.staging.metriport.com", {
```

Comment:
To be made dynamic as part of another PR, there's an item for this on the ticket's TODO: https://github.com/metriport/metriport-internal/issues/228